packages/core/src/api/schema/admin-api/api-key.type.graphql

```diff
@@ -0,0 +1,44 @@
+type ApiKey implements Node {
+    id: ID!
+    createdAt: DateTime!
+    updatedAt: DateTime!
+
+    """
+    ID by which we can look up the API-Key.
+    Also helps you identify keys without leaking the underlying secret API-Key.
+    """
+    lookupId: String!
```

Comment:
Is this something like a "public key"?
Here we could think about a better name imo